### DISCUSSION TITLE
algolia Search for configuration-related issues

### DISCUSSION THREAD

**Discussion by zeMinng** [vuejs/vitepress#3416] [Q&A] [answered]
In the vitepress document, I found the Algolia description, first I need Algolia application. At present, the official sent me a successful email, but it did not send the corresponding AppId, apiKey. What should I do?

This is the Algolia responding to my email
![image](https://github.com/vuejs/vitepress/assets/75962652/911870b9-9e80-4297-aa76-9229e90cf0b5)

**Comment by zeMinng** [comment]
I need to get the AppId in the Algolia. Is the apiKey enough?
Search for the configuration code and just start working?

![image](https://github.com/vuejs/vitepress/assets/75962652/e590139a-e94a-4bd3-9ce5-f1ddb2c65b84)

**Reply by brc-dd** [reply]
Both are needed. They probably have sent older instructions. You can get them from the crawler console - https://crawler.algolia.com/admin/crawlers

It should show something like this:

<img width="1308" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/1e5a4af8-f546-4b5e-b905-65a8b36c9903">

**Reply by zeMinng** [reply]
but，Can t log in normally I need your help thank you
This is normal?
![image](https://github.com/vuejs/vitepress/assets/75962652/bea2cab3-c17c-45f6-94d6-53d4eb16e712)

**Reply by brc-dd** [reply]
Can you try accessing https://dashboard.algolia.com instead?

**Reply by zeMinng** [reply]
ok，but  https://crawler.algolia.com/admin/crawlers    won't do
algolia I was logged in using the github way
![image](https://github.com/vuejs/vitepress/assets/75962652/fac8a2b0-6b3f-4ea6-a1be-36479a3ff7b3)

**Reply by brc-dd** [reply]
Click on API Keys there, it should have your application id and search API key

**Reply by zeMinng** [reply]
Yes, but it doesn't work properly
![image](https://github.com/vuejs/vitepress/assets/75962652/22859027-c5d5-4f5c-9208-de2d4bacc6fb)
![image](https://github.com/vuejs/vitepress/assets/75962652/8c160631-d0f9-4644-ab39-32fc02448efb)
![image](https://github.com/vuejs/vitepress/assets/75962652/c75ccae8-1a68-4238-b7f9-5002551d42e8)

**Reply by brc-dd** [reply]
Can you share the screenshot of https://dashboard.algolia.com/account/applications

**Reply by zeMinng** [reply]
![image](https://github.com/vuejs/vitepress/assets/75962652/e17962ab-7c9e-4891-8c6d-3489b80647e9)

**Reply by brc-dd** [reply]
Ah, I don't think this is the right account you've logged into. This should show an application with plan "DOCSEARCH". From which account did you contact algolia support?

**Reply by zeMinng** [reply]
in the https://docsearch.algolia.com/apply
 There is no option for me to log in

**Reply by brc-dd** [reply]
No but you would've provided an email there, right (when you filled the form for application)? Can you try logging / creating account using that email? (Login/Create account using that email not github/google, etc.)

**Reply by zeMinng** [reply]
Use that email address and log in to the same account
Too disappointed
![image](https://github.com/vuejs/vitepress/assets/75962652/4aa10212-3999-4fcd-84d0-602a4b87ab80)

**Reply by brc-dd** [reply, marked as answer]
I think you should contact Algolia support again and tell them that you’re not seeing docsearch application in dashboard.

**Reply by zeMinng** [reply]
thank you
